content/open-source/concepts/next-js/next-js.md

```diff
@@ -0,0 +1,30 @@
+---
+Title: 'Next.js'
+Description: 'Next.js is an open-source framework that brings the best of React client-side development to the server-side along with built-in features like TypeScript support and route pre-fetching.'
+Subjects:
+  - 'Developer Tools'
+  - 'Web Development'
+Tags:
+  - 'Node'
+  - 'Open Source'
+  - 'React'
+CatalogContent:
+  - 'paths/front-end-engineer-career-path'
+  - 'paths/computer-science'
+---
+
+[**Next.js**](https://nextjs.org) is an open-source development [framework](https://www.codecademy.com/resources/docs/general/framework) that brings the best of [React's](https://www.codecademy.com/resources/docs/react) client-side functionality to the [server-side](https://www.codecademy.com/resources/docs/general/server-side-rendering) when building web applicatons.
+
+Additionally, Next.js comes with built-in features including the following:
+
+- A page-based [routing](https://www.codecademy.com/resources/docs/general/routing) system that supports dynamic routes.
+- Every page is pre-rendered for improved site performance and search-engine optimization (SEO).
+- Any page can either be statically generated (SSG) ahead of time or rendered on the server-side (SSR) upon request.
+- Faster page loads enables automatic code splitting or "lazy-loading".
+- An in-house `Link` component is used for client-side routing with optional pre-fetching.
+- There is vast styling support including built-in [CSS](https://www.codecademy.com/resources/docs/css), Sass, and CSS-in-JS libraries.
+- [TypeScript](https://www.codecademy.com/resources/docs/typescript) is supported in Next.js.
+
+Next.js was originally authored by Guillermo Rauch on [GitHub](https://www.codecademy.com/resources/docs/general/github) in 2016. Due to its functionality, large companies such as Netflix, Starbucks, Uber, GitHub, and Ticketmaster have all used Next.js for their website development. It is currently owned and maintained by the company [Vercel](https://vercel.com).
+
+**Fun Fact:** Codecademy Docs is built with Next.js.
```

Comment:
This is nice! Are we able to do fun facts around Docs and if so, do we have some guidelines for it?